manifests/params.pp

```diff
@@ -1,6 +1,7 @@
 # puma::params class
 class puma::params {
   $puma_user          = 'puma'
+  $www_user           = 'www-data'
```

Comment:
Does doing it this way prevent a client from specifying a different $www_user?  Or will this be overridden if they do?

Reply:
The default values are replaced by custom values if they're specified (we use user and group `vagrant`).